client/src/pages/Edit.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useEffect } from "react";
+import Button from "@mui/material/Button";
+import styled from "styled-components";
+import { useNavigate } from "react-router-dom";
+import axios from "axios";
+
+function Edit() {
+  const navigate = useNavigate();
+  const [editFood, setEditFood] = useState({
+    foodName: "",
+    foodAuthor: "",
+    img: "",
+  });
+
+  useEffect(() => {
+    const editFoodId = async () => {
+      const reqdata = await fetch("http://localhost:5000/food");
+      const response = reqdata.json();
+      console.log(response);
+    };
+    editFoodId();
+  }, []);
+
+  const onInputChange = (e) => {
+    setEditFood({ ...editFood, [e.target.name]: e.target.value });
+  };
+
+  const FormHandle = (e) => {
+    e.preventDefault();
+    const res = axios.post("http://localhost:5000/food", editFood);
+    console.log(res);
+  };
+
+  const navigateToEdit = () => {
+    navigate("/details");
+  };
+
+  return (
+    <div>
+      <Container>
+        <h1>Updated Food Items!</h1>
+        <div>
+          <form onSubmit={(e) => FormHandle(e)}>
```

Comment:
Since `FormHandle` already accepts `e` as an argument, you don't need this anonymous function. You can safely use a function reference here instead.

```suggestion
          <form onSubmit={FormHandle}>
```